### ISSUE TITLE
Review LOI: Proposal

### ISSUE THREAD

**Issue by smiths** [smiths/TCTvMDE#6]
The first draft of the proposal for the LOI is now complete (https://github.com/smiths/TCTvMDE/blob/master/LOI/Proposal.tex).  Please review.  Feel free to make changes directly in the tex source, or to suggest changes as comments in the discussion below.  When you are done your review, please check your name off on the following list:

- [ ] @JacquesCarette 
- [ ] @richpaige 
- [ ] @ZMotamed

**Comment by JacquesCarette** [comment]
Deadline?

**Comment by smiths** [comment]
The final deadline for submission to NFRF is September 2.  Let us set the deadline for your reviews to Monday, August 26.

**Comment by ZMotamed** [comment]
Done.

**Comment by smiths** [comment]
@ZMotamed, you didn't make any changes to the proposal.  Can I assume that this means you did not see anything that you thought required changing?  Assuming that is the case, I'll check your name off on the review list in the original issue.

**Comment by ZMotamed** [comment]
I read and changed HighReward.tex and HighRisk.tex as requested. Can you see them?
I will review the proposal later.

**Comment by smiths** [comment]
Yes, I can see the changes to HighReward.tex and HighRisk.tex.  I'm looking over them now.  My question was because you commented "Done" above for this issue about reviewing the Proposal.  (The wording may be confusing, Proposal is a separate 1 page file, Proposal.tex.)
